index.html

```diff
@@ -6,10 +6,142 @@
+            </p>
+            <div class="location-time">
+                <span class="time" id="time"> </span>
+                <div class="location">Afghanistan, Daikundi, Kity, Temran</div>
+            </div>
+        </div>
+    </section>
+
+    <!-- main program section start -->
+    <section class="main-program" id="main-program">
+        <h1 class="main-program-title">Main program </h1>
+           <span class="horizontal-line"> </span>
+        <div class="program-container">
+            <div class="program-item">
+                <img class="program-img" src="assets/images/lecture-icon.png" alt="program image" >
+                <h2 class="program-title"> Leture </h2>
+                <p class="pro-description">
+                    HTML is the standard markup language for Web pages. With HTML you can create your site.
+                </p>
+            </div>
+
+            <div class="program-item">
+                <img class="program-img" src="assets/images/exhibition-icon.png" alt="program image" >
+                <h2 class="program-title"> Courses </h2>
+                <p class="pro-description">
+                    HTML is the standard markup language for Web pages. With HTML you can create your site.
+                </p>
+            </div>
 
+            <div class="program-item">
+                <img class="program-img" src="assets/images/forum-icon.png" alt="program image" >
+                <h2 class="program-title"> Forum </h2>
+                <p class="pro-description">
+                    HTML is the standard markup language for Web pages. With HTML you can create your site.
+                </p>
+            </div>
+
+            <div class="program-item">
+                <img class="program-img" src="assets/images/workshop-icon.png" alt="program image" >
+                <h2 class="program-title"> Workshop </h2>
+                <p class="pro-description">
+                    HTML is the standard markup language for Web pages. With HTML you can create your site.
+                </p>
+            </div>
+
+            <div class="program-item">
+                <img class="program-img" src="assets/images/ignite-icon.png" alt="program image" >
+                <h2 class="program-title"> CC-ignight </h2>
+                <p class="pro-description">
+                    HTML is the standard markup language for Web pages. With HTML you can create your site.
+                </p>
+            </div>
+        
+        </div>
+        <div class="main-program-link">
+            <button class="join"> JOIN SAZISH ACADEMY  </button>
+            <a class="see-all-program" href="#">  <span class="see-all"> SEE ALL PROGRAM </span>  </a>
+        </div>
+    </section>
+
+    <!-- speaker section start -->
+    <section class="speakers">
+        <h1 class="speaker-headline">  Feature Speakers  </h1>
+           <hr>
```

Comment:
- [ ] Kindly make sure the feature speakers pictures have the same size.
- Your design
<img width="1246" alt="Screenshot 2023-05-31 at 09 34 52" src="https://github.com/akbarsazish/Sazish-Academy/assets/81444338/a3b837a4-fa64-4f47-9a26-3e5a2c966530">

- Behance Design
<img width="1352" alt="Screenshot 2023-05-31 at 05 26 43" src="https://github.com/akbarsazish/Sazish-Academy/assets/81444338/a70297b4-faa8-421b-b27e-b8f74a63c159">